R/prepare-admin-panel.R

```diff
@@ -265,7 +265,7 @@ prepare_admin_panel_components <- function(
 
         semantic.dashboard::valueBox(
           value = time_hours,
-          subtitle = "Average time spent on app daily",
+          subtitle = "Average time spent daily",
```

Comment:
I removed text that I think are quite repetitive, e.g. on app, usage, accessed,  in the box titles.